bin/build

```diff
@@ -2,26 +2,44 @@
 
 set -e
 
-. bin/build_utils
+export DOCKER_BUILDKIT=1
+DEBUG=false
 
-FULL_VERSION_TAG="$(full_version_tag)"
+if [ "$1" == "--debug" ]; then
+    DEBUG=true
+fi
 
-echo "---"
+cd $(dirname $0)/..
 
-echo "Building secrets-provider-for-k8s Docker image"
+. bin/build_utils
 
-docker build \
-  --tag "secrets-provider-for-k8s:dev" \
-  --tag "secrets-provider-for-k8s:${FULL_VERSION_TAG}" \
-  --tag "secrets-provider-for-k8s:latest" \
-  --target "secrets-provider" \
-  .
+FULL_VERSION_TAG="$(full_version_tag)"
 
-echo "Building secrets-provider-for-k8s-redhat:$FULL_VERSION_TAG Docker image"
+echo "---"
 
-docker build --tag "secrets-provider-for-k8s-redhat:${FULL_VERSION_TAG}" \
-   --build-arg VERSION="$FULL_VERSION_TAG" \
-   --target "secrets-provider-for-k8s-redhat" \
-   .
+if [ $DEBUG = true ]; then
+  echo "Building secrets-provider-debug Docker image"
```

Comment:
I understand for backwards compatibility but not sure we need to debug the red hat image because there are no real changes to the code to make it "rh-certified"